docs/send-data/hosted-collectors/http-source/zoom.md

```diff
@@ -0,0 +1,85 @@
+---
+id: zoom
+title: Zoom Source
```

Comment:
```suggestion
title: Zoom HTTP Source
```

Reply:
Going as per the UI. Even though this is a HTTP source UI doesn't mention HTTP.